adb-utils.spec

```diff
@@ -27,6 +27,8 @@ specific service and directly including it to kickstart file.
 
 %{__install} -pm644 services/openshift/openshift.service \
 %{buildroot}%{_unitdir}/openshift.service
+%{__install} -d -m 755 %{buildroot}%{_sysconfdir}/bash_completion.d/
+%{__install} -pm644 bash_completion/* %{buildroot}%{_sysconfdir}/bash_completion.d/
```

Comment:
@praveenkumar  My point was about style consistency of using short options. If we use '-p -m 775' on one line we should not use '-pm755' (group it together) on the other.